store/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,18 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ProductsRoutingModule } from './products-routing/products-routing.module';
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: '',
+    redirectTo: '/products',
+    pathMatch: 'full'
+  },
+  {
+    path: 'products',
+    loadChildren: () => import('./products-routing/products-routing.module').then((m) => ProductsRoutingModule)
```

Comment:
Should be ` .then(m => m.ProductsModule)`

You loading main `ProductsModule`, which includes `ProductsRoutingModule`